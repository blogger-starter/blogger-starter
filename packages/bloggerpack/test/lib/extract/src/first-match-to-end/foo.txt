<example>
foo
</continue>

<example>
foo2
</continue>

<example>
foo3
</end>
