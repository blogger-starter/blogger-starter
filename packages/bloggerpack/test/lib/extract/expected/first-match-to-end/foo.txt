/* src/first-match-to-end/foo.txt */

foo
</continue>

<example>
foo2
</continue>

<example>
foo3
